core/src/components/checkbox/checkbox.scss

```diff
@@ -148,46 +148,53 @@ input {
   opacity: 0;
 }
 
-// Justify Content
-// ---------------------------------------------
+// Checkbox Bottom Content
+// ----------------------------------------------------------------
+
+.checkbox-bottom {
+  @include padding(4px, null, null, null);
```

Comment:
This should not be on the common scope, as each theme will use its proper token (even if the value is the same). 

Maybe we can create now the checkbox.native.scss and divide the Design System decision there (mostly this padding and the font-sizes I think)

Reply:
Spoke with Bernardo offline - this will be accomplished in another task